FreydCategoriesForCAP/gap/CategoryOfRows.gi

```diff
@@ -1048,6 +1051,66 @@ InstallGlobalFunction( INSTALL_FUNCTIONS_FOR_CATEGORY_OF_ROWS,
        
         ##
         AddMorphismToBidualWithGivenBidual( category, { cat, obj, dual } -> IdentityMorphism( cat, obj ) );
+
+        ## Operations related to the cohom-tensor adjunction
+
+        ##
+        AddCoDualOnObjects( category, { cat, obj } -> obj );
+
+        ##
+        AddCoDualOnMorphismsWithGivenCoDuals( category,
+          function( cat, codual_source, morphism, codual_range )
+            
+            return CategoryOfRowsMorphism( cat, codual_source,
+                                           TransposedMatrix( UnderlyingMatrix( morphism ) ),
+                                           codual_range );
+            
+        end );
+        
+        ##
+        AddCoclosedEvaluationForCoDualWithGivenTensorProduct( category,
+          function( cat, tensor_object, object, unit )
+            local rank, id;
+            
+            rank := RankOfObject( object );
+            
+            if rank = 0 then
+                
+                return ZeroMorphism( cat, tensor_object, unit );
```

Comment:
According to the documentation I would expect `ZeroMorphism( cat, unit , tensor_object)`